dashboard/app/views/levels/_free_response.html.haml

```diff
@@ -1,16 +1,33 @@
-:javascript
-  function getResult() {
-    return {
-      response: $('.response').val(),
-      result: true
-    };
-  }
+- level ||= @level
+- last_attempt ||= @last_attempt
+- index ||= nil
+- number ||= nil
+
+- if index
```

Comment:
I think `index` can serve both purposes here: `standalone` is true iff `index` is not set.